tools/lsp/common/rename_component.rs

```diff
@@ -648,67 +648,43 @@ mod tests {
 
     use crate::common::test;
     use crate::common::text_edit;
-    use crate::preview;
-
-    /// Find the identifier that belongs to a struct of the given `name` in the `document`
-    fn find_struct_identifiers(
-        document: &syntax_nodes::Document,
-        name: &str,
-    ) -> Vec<syntax_nodes::DeclaredIdentifier> {
-        let mut result = vec![];
-        let name = i_slint_compiler::parser::normalize_identifier(name);
-
-        for el in document.ExportsList() {
-            for st in el.StructDeclaration() {
-                let identifier = st.DeclaredIdentifier();
-                if i_slint_compiler::parser::normalize_identifier(&identifier.text().to_string())
-                    == name
-                {
-                    result.push(identifier);
-                }
-            }
-        }
-
-        for st in document.StructDeclaration() {
-            let identifier = st.DeclaredIdentifier();
-            if i_slint_compiler::parser::normalize_identifier(&identifier.text().to_string())
-                == name
-            {
-                result.push(identifier);
-            }
-        }
-
-        result.sort_by_key(|i| i.text_range().start());
-        result
-    }
 
-    /// Find the identifier that belongs to a struct of the given `name` in the `document`
-    fn find_enum_identifiers(
-        document: &syntax_nodes::Document,
-        name: &str,
-    ) -> Vec<syntax_nodes::DeclaredIdentifier> {
-        let mut result = vec![];
-
-        for el in document.ExportsList() {
-            for en in el.EnumDeclaration() {
-                let identifier = en.DeclaredIdentifier();
-                if i_slint_compiler::parser::normalize_identifier(&identifier.text().to_string())
-                    == name
-                {
-                    result.push(identifier);
-                }
-            }
-        }
-
-        for en in document.EnumDeclaration() {
-            let identifier = en.DeclaredIdentifier();
-            if identifier.text() == name {
-                result.push(identifier);
+    #[track_caller]
```

Comment:
The function have many unwrap. It may be difficult to distinguish them if they are all reported at the location of the caller. But I guess this is not a problem in practice anyway as it is more important to know what fails rather than why

Reply:
I found it more convenient this way: The test print the processed output and which file that was taken from already and it typically is pretty obvious what I mistyped with that visible :-)